front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -727,6 +750,39 @@ class SettingsCustomizeTab extends PureComponent<
                     </ToggleLabel>
                   </Setting>
                 </WideSectionField>
+
+                {tenant.data.attributes.settings?.events_widget &&
+                  tenant.data.attributes.settings.events_widget.allowed && (
```

Comment:
```suggestion
                {tenant.data.attributes.settings?.events_widget?.allowed && (
```
I think that'd work too, I think it's slightly more elegant but that's quite subjective a statement, feel free to ignore.